.vsts/job.yml

```diff
@@ -102,10 +95,25 @@ jobs:
     - task: securedevelopmentteam.vss-secure-development-tools.build-task-publishsecurityanalysislogs.PublishSecurityAnalysisLogs@2
       displayName: 'Publish Security Analysis Logs'
 
-   # Publish Test Results
-  - ${{ if eq(parameters.name, 'Windows') }}:
+  # At the moment all the tests fail on Linux, so we only run them on Windows and MacOS
+  - ${{ if or(eq(parameters.name, 'Windows'), eq(parameters.name, 'MacOS')) }}:
     - task: PublishTestResults@2
+      condition: always()
       inputs:
         testResultsFormat: 'JUnit' # Options: JUnit, NUnit, VSTest, xUnit, cTest
         testResultsFiles: '**/test-*.xml'
+        publishRunAttachments: true
 
+  - ${{ if eq(parameters.name, 'Windows') }}:
+    - task: PublishPipelineArtifact@0
+      condition: always()
```

Comment:
You don't need condition. You could use that instead of the if block for windows if you want

Reply:
Done